### ISSUE TITLE
std.Thread.Condition: error: no member named 'srwlock' in struct 'std.Thread.Mutex'

### ISSUE THREAD

**Issue by jeffkdev** [ziglang/zig#9077]
semaphore.zig
```
const std = @import("std");
pub fn main() !void {
    var sem = std.Thread.Semaphore{};
    sem.wait();
}
```

`zig build-exe semaphore.zig`
```
\zig\build-release\lib\zig\std\Thread\Condition.zig:57:19: error: no member named 'srwlock' in struct 'std.Thread.Mutex'
            &mutex.srwlock,
```

OS: Windows 10
Zig version: 0.9.0-dev.70+a3f7a48d9

I also noticed running the same code on Godbolt 0.8.0 gives a different error:
```
<Compilation failed>
# Compiler exited with result code 1
Standard error:
/opt/compiler-explorer/zig-0.8.0/lib/std/Thread/Condition.zig:154:14: error: no member named 'unlock' in struct 'std.Thread.Mutex'
        mutex.unlock();
             ^
```

**Comment by LemonBoy** [comment]
Identical to #8051, is using the search function that hard?

**Comment by jeffkdev** [comment]
I searched for 'srwlock' (the missing field in my error) first and then didn't find anything. I also searched for searched 'semaphore' and didn't find any open issues, just this closed issue https://github.com/ziglang/zig/issues/8052 from February. At that point I thought the issue was happening again.  

Right before posting it I figured having the bug title be about semaphores was less useful than posting the exact the error message instead. Which of course in hindsight if I searched *that* string I would have found the one you linked.

Between `is using the search function that hard?` and `You must really hate the search function because this is a duplicate of...` I'm not sure the hostility is deserved. But thanks the for the help regardless.
